schedule/autoyast_mini_staging.yaml

```diff
@@ -0,0 +1,20 @@
+name:           autoyast_mini_staging@64bit-staging
+description:    >
+    Test verifies installation with minimal autoyast profile. Same as autoyast_mini but with product defined using url and not as registered modules. 
+    Maintainer: riafarov@suse.com
+schedule:
+    - autoyast/prepare_profile
+    - installation/bootloader
+    - autoyast/installation
+    - autoyast/console
+    - autoyast/login
+    - console/yast2_clone_system
```

Comment:
is it possible to become a conditional and have a unified file here as well as the approach to the other schedulers?

Reply:
I guess it is possible to use some conditions over here. Not really sure what we want to achieve. 
Particularly for AY staging, we can have one yaml schedule file for sle12 and sle15, but we need to always attach the corresponding AY profile. Thus we can have plain schedule yaml for both staging projects and keep all variables in openQA for each test suite.

Reply:
So for staging, we can live with 1 yaml and multiple test suites with specific AY profile